public/index.html

```diff
@@ -1,14 +1,18 @@
 <!doctype html>
 
 <html lang="en">
-  <head>
+<head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
     <meta name="theme-color" content="#3498db">
     <link rel="manifest" href="/pwa.json">
     <link rel="shortcut icon" href="/icons/128.png">
+    <script defer src="https://use.fontawesome.com/releases/v5.0.9/js/all.js" integrity="sha384-8iPTk2s/jMVj81dnzb/iFR2sdA7u06vHJyyLlAd4snFpCl/SnyUjRrbdJsw1pGIl" crossorigin="anonymous"></script>
```

Comment:
I bundle all dependencies into the build via webpack, for initial load speed and privacy reasons. I can tweak the bundling with webpack after merging for you.